### PR TITLE
chore(coordinator): integrate generator for create

### PR DESCRIPTION
**Pull request by jayzhang**

### CODE REVIEW

File: packages/fx-core/src/component/coordinator/index.ts [hunk 1]
```diff
@@ -44,15 +89,117 @@ export class Coordinator {
...
+        if (feature === TabOptionItem.id) {
+          templateName = TemplateNames.SsoTab;
+        } else {
+          templateName = TemplateNames.Tab;
+        }
+      }
+      if (templateName) {
+        const langKey = convertToLangKey(language);
+        try {
+          await Generator.generateTemplate(templateName, langKey, projectPath, context);
```
Comment (hund030) [inline packages/fx-core/src/component/coordinator/index.ts:178]: `generateTemplate` returns a Result<> object. Please handle the possible error.

File: packages/fx-core/src/component/coordinator/index.ts [hunk 2]
```diff
@@ -44,15 +89,117 @@ export class Coordinator {
...
+        [TelemetryProperty.Feature]: feature,
+      });
+    }
+
+    // generate unique projectId in projectSettings.json
+    const projectSettingsRes = await loadProjectSettingsByProjectPath(projectPath, true);
+    if (projectSettingsRes.isOk()) {
+      const projectSettings = projectSettingsRes.value;
+      projectSettings.projectId = inputs.projectId ? inputs.projectId : uuid.v4();
+      projectSettings.isFromSample = true;
```
Comment (hund030) [inline packages/fx-core/src/component/coordinator/index.ts:194]: Always true?

**Review by LongOddCode — DISMISSED** [review]

**Review by hund030 — DISMISSED** [review]

File: packages/fx-core/src/component/provisionUtils.ts [hunk 3]
```diff
@@ -208,7 +208,47 @@ export class ProvisionUtils {
     }
     return ok(undefined);
   }
-
+  // /**
```
Comment (1yefuwang1) [inline packages/fx-core/src/component/provisionUtils.ts:211]: why not remove the code

Comment (jayzhang) [reply packages/fx-core/src/component/provisionUtils.ts:211]: Will be used in the future when integrating provision

**Review by LongOddCode — APPROVED** [review]

**Pull request merged**